doc/cephfs/mount.rst

```diff
@@ -0,0 +1,69 @@
+Mounting CephFS
+===============
+
+You can use CephFS by mounting it to your local filesystem or by using
+cephfs-shell. CephFS can be mounted using kernel as well as using FUSE. Both
+have their own advantages. Read the following section to understand more
+about both of these ways to mount CephFS.
+
+Which CephFS Client?
+--------------------
+
+The FUSE client is the most accessible and the easiest to upgrade to the
+version of Ceph used by the storage cluster, while the kernel client will
+often give better performance.
+
+The clients do not always provide equivalent functionality, for example
+the fuse client supports client-enforced quotas while the kernel client
+does not.
+
+When encountering bugs or performance issues, it is often instructive to
+try using the other client, in order to find out whether the bug was
+client-specific or not (and then to let the developers know).
+
+General Pre-requisite for Mounting CephFS
+-----------------------------------------
+Before mounting CephFS, ensure that the client host (where CephFS has to be
+mounted and used) has a copy of the Ceph configuration file (i.e.
+``ceph.conf``) and a keyring of the CephX user that has CAPS for a Ceph MDS.
```

Comment:
`s/CAPS for a Ceph MDS/permission to access the MDS/`

Reply:
Reworded.